trace/widgets/control_panel.py

```diff
@@ -643,6 +644,19 @@ def clear_curves(self) -> None:
                 item.close()
 
     def close(self) -> bool:
+        # Pop up confirming axis delete
+        dialog = QMessageBox(
+            text=str("Are you sure you want to delete the axis?"),
+            parent=self,
+        )
+        dialog.setIcon(QMessageBox.Information)
+        dialog.setWindowTitle("Delete Axis")
+        dialog.setStandardButtons(QMessageBox.Ok | QMessageBox.Cancel)
+        result = dialog.exec_()
+
+        if result == QMessageBox.Cancel:
```

Comment:
```suggestion
        dialog = QtWidgets.QMessageBox(
            text=str("Are you sure you want to delete the axis?"),
            parent=self,
        )
        dialog.setIcon(QtWidgets.QMessageBox.Information)
        dialog.setWindowTitle("Delete Axis")
        dialog.setStandardButtons(QtWidgets.QMessageBox.Ok | QtWidgets.QMessageBox.Cancel)
        result = dialog.exec_()

        if result == QtWidgets.QMessageBox.Cancel:
```

If we remove the import of `QMessageBox` above, then we need to reference it as `QtWidgets.QMessageBox`